vulnfeeds/cves/versions.go

```diff
@@ -62,7 +62,7 @@ func extractGitHubCommit(link string) *FixCommit {
 	}
 }
 
-func ExtractVersionInfo(cve CVEItem) VersionInfo {
+func ExtractVersionInfo(cve CVEItem, validVersions []string) VersionInfo {
```

Comment:
is validVersions meat to be used later? (if so, TODO?)

Reply:
Yep! I had some TODOs already in this function but I'll clarify them to mention this variable explicitly.